### ISSUE TITLE
%group% not correctly being parsed anymore

### ISSUE THREAD

**Issue by NaTorOG** [NEZNAMY/TAB#1221]
### Server version

1.19.4

### TAB version

4.1.2 | Tab Bridge Version 5.0.2

### Describe the bug

In previous TAB versions, %group% was being parsed correctly for UpperCases and LowerCases, now its parsed fully lowercased.

### Expected behavior

%group% being parsed correctly matching the groups.yml file with Uppercases letters 

### Steps to reproduce

TAB on BungeeCord
TAB Bridge in the lobbies
![image](https://github.com/NEZNAMY/TAB/assets/4592301/d1a5faec-6d5a-447b-bdc7-d200b70d6282)

### Additional info

per-server:
  Lobby*:
    Owner:
      tabprefix: '&x&a&5&2&f&c&e▪ ᴏᴡɴᴇʀ ▪&f '
      tagprefix: '#a52fce▪ ᴏᴡɴᴇʀ ▪&f '
      tabsuffix: ''
    Admin:
      tabprefix: '&x&5&4&c&4&c▪ ᴀᴅᴍɪɴ ▪&f '
      tagprefix: '#f54c4c▪ ᴀᴅᴍɪɴ ▪&f '
      tabsuffix: ''
    Dev:
      tabprefix: '&x&4&d&7&f&f&5▪ ᴅᴇᴠ ▪&f '
      tagprefix: '#4d7ff5▪ ᴅᴇᴠ ▪&f '
      tabsuffix: ''
    SrBuilder:
      tabprefix: '&x&d&1&5&6&3&b▪ sʀ.ʙᴜɪʟᴅᴇʀ ▪&f '
      tagprefix: '#d1563b▪ sʀ.ʙᴜɪʟᴅᴇʀ ▪&f '
      tabsuffix: ''
    SrMod:
      tabprefix: '&x&1&b&e&4&d&8▪ sʀ.ᴍᴏᴅ ▪&f '
      tagprefix: '#1be4d8▪ sʀ.ᴍᴏᴅ ▪&f '
      tabsuffix: ''
    Mod:
      tabprefix: '&x&9&8&d&0&f&f▪ ᴍᴏᴅ ▪&f '
      tagprefix: '#98d0ff▪ ᴍᴏᴅ ▪&f '
      tabsuffix: ''
    Helper:
      tabprefix: '&x&6&2&e&4&a&3▪ ʜᴇʟᴘᴇʀ ▪&f '
      tagprefix: '#62e4a3▪ ʜᴇʟᴘᴇʀ ▪&f '
      tabsuffix: ''
    Trainee:
      tabprefix: '&x&7&0&6&b&6&b▪ ᴛʀᴀɪɴᴇᴇ ▪&f '
      tagprefix: '#706b6b▪ ᴛʀᴀɪɴᴇᴇ ▪&f '
      tabsuffix: ''    
    Donatore:
      tabprefix: '&x&d&8&a&e&3&2▪ ᴅᴏɴᴀᴛᴏʀᴇ ▪&f '
      tagprefix: '#d8ae32▪ ᴅᴏɴᴀᴛᴏʀᴇ ▪&f '
      tabsuffix: ''
    default:
      tabprefix: ''
      tagprefix: ''
      tabsuffix: ''

### Checklist

- [X] I am running latest version of the plugin
- [X] I have read the wiki to make sure it's not an issue with configuration
- [ ] I ticked all of the boxes without actually reading them
- [X] I have read the Compatibility wiki page and am not trying to run the plugin on an unsupported server version / platform
- [X] I am willing to cooperate to get this issue resolved faster, including providing any additional information requested or testing new builds

**Comment by NEZNAMY** [comment]
Previously I was using Vault API for getting groups from LP in bridge, but it turns out LuckPerms' Vault permission implementation returns group's display name instead of actual name, which was inconsistent with the main plugin, which is and always was using a direct LP hook. Display names often contain various garbage making it hard to work with, such as color codes. I also think LP user info command does not show group's display name, which caused a lot of confusion to both me and users asking for help.

In conclusion, bridge finally works as it should have always worked. See `/lp user <player> info`, that's the group the plugin is supposed to take and work with.

**Comment by NaTorOG** [comment]
> Previously I was using Vault API for getting groups from LP in bridge, but it turns out LuckPerms' Vault permission implementation returns group's display name instead of actual name, which was inconsistent with the main plugin, which is and always was using a direct LP hook. Display names often contain various garbage making it hard to work with, such as color codes. I also think LP user info command does not show group's display name, which caused a lot of confusion to both me and users asking for help.
> 
> In conclusion, bridge finally works as it should have always worked. See `/lp user <player> info`, that's the group the plugin is supposed to take and work with.
> 
> The %group% placeholder itself is used mostly internally and can be used for technical purposes, it's not supposed to be a replacement for prefix or anything like that.

I understand the situation, thanks for the help provided.

I used %group% for the previous problem of long placeholders not being parsed sometimes

**Comment by NEZNAMY** [comment]
That was fixed already in latest bridge.
